### PR TITLE
[Tests-Only] Add tests for adding and listing groups

### PR DESCRIPTION
**Pull request by dpakach**
Add tests for HTTP api for
- Listing groups
- Creating new group (not implemented)
- Listing members in a group
- Delete Group

### CODE REVIEW

**Comment by ownclouders** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- pkg/server/http/svc_test.go  59
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/ocis-ocs/pullRequest?prid=6193950&bid=20208252)

**Comment by ownclouders** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- pkg/server/http/svc_test.go  59
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/ocis-ocs/pullRequest?prid=6193950&bid=20208252)

File: pkg/server/http/svc_test.go [hunk 1]
```diff
@@ -1671,6 +1768,344 @@ func TestRemoveUserFromGroup(t *testing.T) {
...
+					"GET",
+					fmt.Sprintf("/%v/cloud/groups/%v%v", ocsVersion, group, formatpart),
+					"",
+					"admin:admin",
+				)
+
+				if err != nil {
+					t.Fatal(err)
+				}
+				fmt.Println(res.Body.String())
```
Comment (kiranparajuli589) [inline pkg/server/http/svc_test.go]:
```suggestion
```

File: pkg/server/http/svc_test.go [hunk 2]
```diff
@@ -1671,6 +1768,344 @@ func TestRemoveUserFromGroup(t *testing.T) {
...
+				if format == "json" {
+					if err := json.Unmarshal(res.Body.Bytes(), &response); err != nil {
+						t.Fatal(err)
+					}
+				} else {
+					if err := xml.Unmarshal(res.Body.Bytes(), &response.Ocs); err != nil {
+						t.Fatal(err)
+					}
+				}
+
```
Comment (kiranparajuli589) [inline pkg/server/http/svc_test.go:2049]: can you also do `assertStatusCode` here?

**Comment by ownclouders** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- pkg/server/http/svc_test.go  66
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/ocis-ocs/pullRequest?prid=6193950&bid=20208252)

**Comment by phil-davis** [comment]
@dpakach this code is moving to https://github.com/owncloud/mono/tree/master/ocs anyway, so you may as well make a PR there.

There is no point hoping that drone comes back for this repo.

**Review by jasson99 — COMMENTED** [review]
LGTM

**Comment by dpakach** [comment]
Moved to https://github.com/owncloud/ocis/pull/636

**Pull request closed without merging**